app/src/util/queue.h

```diff
@@ -2,8 +2,6 @@
 #ifndef QUEUE_H
 #define QUEUE_H
 
-#include "common.h"
```

Comment:
`common.h` is willingly included in all source files, at least to always correctly load defines from `config.h` and `compat.h`. This is a convention I introduced to avoid this kind of error: 068253a3a299a60d78c7c1b0e760f4bbe3b75bc1